File: JavaScript/01.JavaScript-Basic-February-2021/05.conditionalStatementsAdvanced-Lab/07.workingHours.js
function workingDay(input) {
    let hour = Number(input[0]);
    let day = input[1];

    if ((hour >= 10 && hour <= 18) && day != "Sunday") {
        console.log("open");
    } else {
        console.log("closed");
    }
}

workingDay(["11", "Monday"]);
workingDay(["19", "Friday"]);
workingDay(["11", "Sunday"]);

File: Go/01.Go-Basic-February-2022/05.conditionalStatementAdvanced-Lab/07.workingHours.go
package main

import "fmt"

func main() {
	var hour int
	fmt.Scanln(&hour)

	var day string
	fmt.Scanln(&day)

	if hour < 10 || hour > 18 || day == "Sunday" {
		fmt.Print("closed")
	} else {
		fmt.Print("open")
	}
}
